website/pages/how-to-guides/query-aws-cost-and-usage-athena.mdx

```diff
@@ -0,0 +1,180 @@
+
+![Report Content Options for Resource IDs and Data Refresh](/images/how-to-guides/query-aws-cost-and-usage-athena/athena-int.png)
+
+After verifying the settings of the Cost and Usage Report, select create report!
+
+Please see [here](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cur/put-report-definition.html) for the CLI equivalent.
+
+AWS's `billingreports.amazonaws.com` will deliver these reports to our new S3 Bucket.  Keep in mind that it may take up to 24 hours for AWS to start delivering reports to the Amazon S3 bucket.  If you are creating a fresh S3 bucket, we suggest creating the bucket and coming back the next day to complete the rest of this guide!  Storing the billing reports data in Amazon S3 will incur standard [S3 costs](https://docs.aws.amazon.com/cur/latest/userguide/billing-cur-limits.html).  
+
+You should see multiple files in the S3 bucket once the cost and usage reports have been successfully delivered to your S3 bucket. In the billing console, the cost and usage report created will also show a time stamp for `Data last refreshed` such as the screenshot below.
+
+![Cost and Usage Reports in the Billing Console](/images/how-to-guides/query-aws-cost-and-usage-athena/jason-demo-cur.png)
+
+### 3. Set up Amazon Athena for Cost and Usage Reports
+
+Now that we have successfully created an S3 Bucket to receive Cost and Usage Reports, we will now setup Athena for querying our Cost and Usage Reports.
+
+For this step, we'll use the CloudFormation template that [AWS generates](https://docs.aws.amazon.com/cur/latest/userguide/use-athena-cf.html) in the bucket we previously created for Cost and Usage Reports.  [Athena can be setup manually](https://docs.aws.amazon.com/cur/latest/userguide/cur-ate-manual.html) if CloudFormation does not work for your usage.
+
+```bash copy
+aws cloudformation create-stack \
+--stack-name cost-and-usage-crawler \
+--template-url https://cq-demo-cur.s3.amazonaws.com/cost-and-usage/jason-demo-reports/crawler-cfn.yml \
+--capabilities CAPABILITY_IAM 
+--region us-east-1
+```
+
+Remember to replace the `template-url` parameter with the object URL for the CloudFormation template that AWS created in the Cost and Usage S3 Bucket.  
+
+This template creates the following resources: 
+* 3 IAM Roles
+* An AWS Glue Database
+* An AWS Glue crawler
+* 2 Lambda functions
+* An Amazon S3 notification
+
+As needed, adjust the CloudFormation template depending on your organizational requirements such as required IAM controls, tags, and encryption settings.
+
+You should now see 2 databases loaded in Amazon Athena:
+* CloudQuery Data
+* AWS Cost and Usage Reports
+
+We can run the following query [to validate that Athena is ready](https://docs.aws.amazon.com/cur/latest/userguide/cur-ate-run.html) for queries on the Cost and Usage data.  A status of `READY` means we can query the Athena database.  If the status is `UPDATING`, Athena may return incomplete results.
```

Comment:
```suggestion
We can run the following query [to validate that Athena is ready](https://docs.aws.amazon.com/cur/latest/userguide/cur-ate-run.html) for queries on the Cost and Usage data, replacing`athenacurcfn_jason_demo_reports` with your own database name.  A status of `READY` means we can query the Athena database.  If the status is `UPDATING`, Athena may return incomplete results.
```

Reply:
Aside: we may want to just do search+replace on `jason` and replace it with `cloudquery` for the table names, so it's a little more generalizable :)

Reply:
Yeah!  Good call - I named the tables with jason to avoid collisions.